packages/e2e/features/ui/components/liveness/with-custom-components copy.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Liveness with Custom Components
+
+  Test Liveness with custom components
```

Comment:
In the description, we usually describe what the feature is that we're testing (e.g. Liveness supports custom credential provider <more context...>).